DataConnectors/SymantecProxySG/Connector_Syslog_SymantecProxySG.json

```diff
@@ -0,0 +1,122 @@
+{ 
+    "id": "SymantecProxySG",
+    "title": "Symantec ProxySG", 
+    "publisher": "Symantec", 
+    "descriptionMarkdown": "The [Symantec ProxySG](https://www.broadcom.com/products/cyber-security/network/gateway/proxy-sg-and-advanced-secure-gateway) allows you to easily connect your Symantec ProxySG logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigations. Integrating Symantec ProxySG with Azure Sentinel provides more visibility into your organization's network proxy traffic and will enhance security monitoring capabilities.", 
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "Symantec", 
+         "baseQuery": "SymantecProxySG" 
+     } 
+    ], 
+    "sampleQueries": [ 
+        { 
+        "description" : "Top 10 Denied Users", 
+        "query": "SymantecProxySG \n | where sc_filter_result == 'DENIED' \n | summarize count() by cs_userdn \n | top 10 by count_"     
+        },
+        {
+        "description" : "Top 10 Denied Client IPs",
+        "query": "SymantecProxySG \n | where sc_filter_result == 'DENIED' \n | summarize count() by c_ip \n | top 10 by count_" 
+        }
+    ], 
+    "dataTypes": [ 
+    { 
+        "name": "Syslog (SymantecProxySG)", 
+        "lastDataReceivedQuery": "SymantecProxySG \n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
+    } 
+], 
+        "connectivityCriterias": [ 
+        { 
+            "type": "IsConnectedQuery", 
+            "value": [ 
+            "SymantecProxySG \n      | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)" 
+        ] 
+        } 
+    ], 
+    "availability": { 
+    "status": 1 
+    }, 
+    "permissions": { 
+    "resourceProvider": [ 
+            { 
+                "provider": "Microsoft.OperationalInsights/workspaces", 
+                "permissionsDisplayText": "write permission is required.", 
+                "providerDisplayName": "Workspace", 
+                "scope": "Workspace", 
+                "requiredPermissions": { 
+                "write": true, 
+                "delete": true 
+            } 
+        } 
+    ],
+    "customs": [
+        {
+            "name": "Symantec ProxySG",
+            "description": "must be configured to export logs via Syslog"
+        }
+    ]
+}, 
+    "instructionSteps": [ 
+        { 
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://raw.githubusercontent.com/chicduong/Azure-Sentinel/acn_cd/Parsers/SymantecProxySG/SymantecProxySG.txt) to create the Kusto functions alias, **SymantecProxySG**",
```

Comment:
Please add a link to the parser as https://aka.ms/sentinelgithubparserssymantecproxysg - currently I've linked to the same link you have mentioned and once I merge the PR, I'll update the aka.ms link with the right Parser path in master. 
This aka.ms link makes this robust from GitHub folder changes requiring a data connector update.

Reply:
Corrections made and pushed